newIDE/app/src/PlatformSpecificAssetsEditor/PlatformSpecificAssetsDialog.js

```diff
@@ -144,6 +150,15 @@ export default class PlatformSpecificAssetsDialog extends React.Component<
             }
           )
         ),
+        resizeImage(
+          fullPath,
+          path.join(projectPath, 'android-windowSplashScreenAnimatedIcon.png'),
+          {
+            width: 576,
```

Comment:
I assume this is the required size?

Reply:
I've added a comment and actually bumped this size so it looks good on a XXDPI screen.